person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age? || parent_permission
+  end
```

Comment:
- [ ] On line `21` you have to use the instance variable `@parent_permission` to work.

 - [ ] Instead of writing `public` you can use this method(`can_use_services?`) after line 10 directly since by default the scope of the method is public.

Reply:
Thank you for your feedback. I implement all the changes said by you.